### DISCUSSION TITLE
focusOnHover default value on SelectItem vs ComboboxItem

### DISCUSSION THREAD

**Discussion by georgekaran** [ariakit/ariakit#1827] [Questions] [answered]
I'm currently building a `Select-combobox`, and as I was taking a look at Ariakit's codebase when I noticed that `SelectItem` has by default `focusOnHover` as `true`, while `ComboboxItem` is `false`.

Is there a reason behind not having the same behavior on both components? 

I'm asking this because while I was composing my component I'm getting two behaviors when using a `Select` vs `Select-combobox`

**Comment by diegohaz** [comment, marked as answer]
There's no special reason. We can change it to `true` by default (before the final v2.0.0 release) if it's more common (looks like it is).

**Reply by georgekaran** [reply]
I agree with you. 

I think it's better to have the same behavior for `SelectItem` and `ComboboxItem` regarding `focusOnHover`.

If you're okay with that, I can open a PR with this change.

**Reply by diegohaz** [reply]
Sure! Feel free to submit a PR. You may have to change some examples/tests, though. It's important that the default examples use the default prop value, so the tests may have to be modified in that case. But we need at least one example with `focusOnHover={false}` to ensure it works.
